jobs/integr8ly/ocp3/nightly/osd-testing-nightly-trigger.yaml

```diff
@@ -7,7 +7,7 @@
     sandbox: false
     concurrent: false
     triggers:
-      - timed: "H(20) 3 * * *"
+      - timed: "20 3 * * *"
```

Comment:
I also added fix for time definition in the new osd nightly pipeline since the original is wrong.